cmip6_downscaling/config.py

```diff
@@ -24,7 +24,7 @@
         "cloud": {
             "storage_options": {'container': 'prefect'},
             "agent": "az-eu-west",
-            "extra_pip_packages": "git+https://github.com/carbonplan/cmip6-downscaling.git git+https://github.com/pangeo-data/scikit-downscale.git",
+            "extra_pip_packages": "git+https://github.com/carbonplan/cmip6-downscaling.git@bugfix/xesmf git+https://github.com/pangeo-data/scikit-downscale.git",
```

Comment:
just to be sure, you won't merge this change into main, right? i want to make sure that our cloud deployment still uses main (and that this change was only so that you could confirm that `bugfix/xesmf` works when testing)